ccw.core/src/clj/ccw/eclipse.clj

```diff
@@ -489,7 +489,7 @@
                        (-> root-path (.append path))
                        (-> root-path (.append package-path) (.append path)))
                 search (workspace-resource path)]
-          :when search]
+          :when (and search (->> search (instance? org.eclipse.core.internal.resources.Folder) not))]
```

Comment:
When possible(*), we do not depend on internal Eclipse packages. The fact that we are in presence of an Eclipse IFile is idiomatically checked through the `IResource` interface like this: `(= org.eclipse.core.resources.IResource/FILE (.getType search))`. Can you change it please?


(*): it's not always possible